### DISCUSSION TITLE
Can we have a discord server where we can help each other about this package?

### DISCUSSION THREAD

**Discussion by clydesantiago** [osiset/laravel-shopify#1217] [General]
Can we have a discord server where we can help each other about this package?

**Comment by Kyon147** [comment]
@osiset what do you think?

**Reply by gnikyt** [reply]
I'm totally fine with it. I barely use Discord to be honest, only to talk to a family member, so I am not sure the procedure of setting it up.

**Reply by Kyon147** [reply]
NP. I'll look at getting one set up and add you to it.

**Comment by Kyon147** [comment]
Here's a server link to a new discord server https://discord.gg/3ZwVrEUjeR

This link never expires but is limited to 100 users - just for spam protection.
